examples/microprofile/hello-world-explicit/pom.xml

```diff
@@ -41,7 +41,7 @@
     <dependencies>
         <dependency>
             <groupId>io.helidon.microprofile.bundles</groupId>
-            <artifactId>helidon-microprofile-3.1</artifactId>
+            <artifactId>helidon-microprofile-core</artifactId>
```

Comment:
This should be full microprofile as well.

Reply:
One benefit of doing what you suggest is that users can then use any of the MP technologies if they use the example `pom`, without having to change which bundle they reference.

The cost is the possible confusion as to why the example - which actually only requires `core` - declares a dependency on the full bundle.

I assume you are OK with that trade-off.

Reply:
This is a basic example that shows how to start MP. I expect people use example as a first step to use Helidon - e.g. it is expected that additional MP specs will be used.